### PR TITLE
chore: remove deprecated method warnings

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

**Review by athakor — APPROVED** [review]
LGTM

**Review by dmitry-fa — APPROVED** [review]
Looks good

File: google-cloud-translate/src/test/java/com/google/cloud/translate/TranslateOptionsTest.java [hunk 1]
```diff
@@ -18,18 +18,19 @@
...
   public void testInvalidTransport() {
-    thrown.expect(IllegalArgumentException.class);
-    TranslateOptions.newBuilder()
-        .setTransportOptions(EasyMock.<TransportOptions>createMock(TransportOptions.class));
+    try {
+      TranslateOptions.newBuilder()
+          .setTransportOptions(EasyMock.<TransportOptions>createMock(TransportOptions.class));
+      Assert.fail();
+    } catch (IllegalArgumentException ex) {
+      Assert.assertNotNull(ex.getMessage());
```
Comment (rahulKQL) [inline google-cloud-translate/src/test/java/com/google/cloud/translate/TranslateOptionsTest.java:33]: Please be consistent with rest of the class, Import `Assert`

File: google-cloud-translate/src/test/java/com/google/cloud/translate/OptionTest.java [hunk 2]
```diff
@@ -47,14 +43,13 @@ public void testHashCode() {
     assertEquals(OPTION.hashCode(), OPTION_EQUALS.hashCode());
   }
 
-  @Test
+  @Test(expected = NullPointerException.class)
```
Comment (rahulKQL) [inline google-cloud-translate/src/test/java/com/google/cloud/translate/OptionTest.java:46]: It be great to use try, catch block for this, similar to below classes

**Review by rahulKQL — APPROVED** [review]

**Pull request closed without merging**